src/solvers/prop/prop_conv_solver.cpp

```diff
@@ -365,7 +365,7 @@ bool prop_conv_solvert::set_equality_to_true(const equal_exprt &expr)
   return true;
 }
 
-void prop_conv_solvert::set_to(const exprt &expr, bool value)
+void prop_conv_solvert::do_set_to(const exprt &expr, bool value)
```

Comment:
Give this a better name, or provide some documentation to advise a future editor on whether they want `set_to` or `do_set_to`

Reply:
documentation extended